lib/Runtime/Library/JavascriptObject.cpp

```diff
@@ -858,10 +859,12 @@ namespace Js
 
         for (uint i = 0; i < ownPropsArray->GetLength(); i++)
         {
-            ownPropsArray->DirectGetItemAt(i, &propKey);
+            BOOL getPropResult = ownPropsArray->DirectGetItemAt(i, &propKey);
+            Assert(getPropResult);
```

Comment:
How should we handle this case in Release builds (when the Assert is compiled out)? Is `getPropResult == false` something that should **never** happen if the code is written correctly?